packages/main/src/plugin/telemetry/telemetry.ts

```diff
@@ -308,7 +304,7 @@ export class Telemetry {
     if (!this.telemetryEnabled) {
       return;
     }
-    this.internalTrack(event, eventProperties).catch((err: unknown) => {
+    void this.internalTrack(event, eventProperties).catch((err: unknown) => {
```

Comment:
I don't see here why you need to add `void`

Reply:
That is correct. It is not needed. Removed.